test/it/parser/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -1242,4 +1242,163 @@
             </expression-projection>
         </projections>
     </select>
+
+    <select sql-case-id="select_row_number">
+        <projections start-index="7" stop-index="25">
+            <expression-projection text="ROW_NUMBER() OVER w" start-index="7" stop-index="25">
+                <expr>
+                    <function function-name="ROW_NUMBER" start-index="7" stop-index="25" text="ROW_NUMBER() OVER w" />
+                </expr>
+                <window start-index="22" stop-index="34">
+                    <window-item start-index="27" stop-index="34">
+                        <order-by>
+                            <column-item name="user_id" order-direction="ASC" start-index="32" stop-index="34"/>
```

Comment:
Where does the user_id column come from? I did not find this field in SQL 
```sql
SELECT ROW_NUMBER() OVER w FROM t_order
```

Reply:
yes , their is mistake in test case. Will correct.